README.md

```diff
@@ -34,3 +34,11 @@ make[1]: Leaving directory '.../linux-with-rust-support'
 For details about the Rust support, see https://github.com/Rust-for-Linux/linux.
 
 For details about out-of-tree modules, see https://www.kernel.org/doc/html/latest/kbuild/modules.html.
+
+## Rust-analyzer
```

Comment:
I think they don't capitalize it:

```suggestion
## rust-analyzer

```

Also please add an empty line (we do the same for the title).